odoo/tests/common.py

```diff
@@ -334,9 +334,23 @@ def _compare_candidate(record, candidate):
                     return False
             return True
 
+        def _repr_field_value(record, field_name):
+            record_value = record[field_name]
+            field_type = record._fields[field_name].type
+            if field_type == 'monetary':
+                currency_field_name = record._fields[field_name].currency_field
+                record_currency = record[currency_field_name]
+                return record_currency and record_currency.amount_to_text(record_value) or record_value
```

Comment:
`amount_to_text` ???

Reply:
Right, `round` is probably closer to what we are looking for (replicate `record_currency.compare_amounts(candidate_value, record_value)`)